test/acceptance/features/serviceClass.feature

```diff
@@ -0,0 +1,88 @@
+Feature: Use ServicesClass resources to manage RegisteredService resources
+
+    Background:
+        Given Primaza Cluster "primaza-main" is running
+        And   Resource "backend_crd.yaml" is installed on "primaza-main"
+
+    Scenario: A Service Class creates Registered Services as specified
+        Given On Primaza Cluster "primaza-main", Resource is created
+            """
+            apiVersion: stable.example.com/v1
+            kind: Backend
+            metadata:
+                name: $scenario_id-1
+            spec:
+                host: internal.db.stable.example.com
+            """
+        And   On Primaza Cluster "primaza-main", Resource is created
+            """
+            apiVersion: stable.example.com/v1
+            kind: Backend
+            metadata:
+                name: $scenario_id-2
+            spec:
+                host: external.db.stable.example.com
+            """
+
+        When  On Primaza Cluster "primaza-main", Resource is created
+            """
+            apiVersion: primaza.io/v1alpha1
+            kind: ServiceClass
+            metadata:
+                name: $scenario_id-serviceclass
+            spec:
+                constraints: {}
+                resource:
+                    apiVersion: stable.example.com/v1
+                    kind: Backend
+                    serviceEndpointDefinitionMapping:
+                      - name: host
+                        jsonPath: .spec.host
+                serviceClassIdentity:
+                  - name: type
+                    value: backend
+                  - name: provider
+                    value: stable.example.com
+                  - name: version
+                    value: v1
+            """
+        Then RegisteredService.primaza.io/$scenario_id-1 is available in cluster "primaza-main"
+        And  jsonpath ".spec.serviceEndpointDefinition[0]" on "registeredservices.primaza.io/$scenario_id-1" in cluster primaza-main is "{"name":"host","value":"internal.db.stable.example.com"}"
+        And  RegisteredService.primaza.io/$scenario_id-2 is available in cluster "primaza-main"
+        And  jsonpath ".spec.serviceEndpointDefinition[0]" on "registeredservices.primaza.io/$scenario_id-2" in cluster primaza-main is "{"name":"host","value":"external.db.stable.example.com"}"
+        And  The resource serviceclasses.primaza.io/$scenario_id-serviceclass is deleted from the cluster "primaza-main"
```

Comment:
If this scenario fails before executing this cleanup, other would be affected.  We may use a fixture for this.

Reply:
As it stands right now, we delete & remake the cluster between each scenario, so this won't be impacted.  If we move away from that model, we'll need to update this.